profile/src/main/resources/profileschema/0.1/datahelix.schema.json

```diff
@@ -79,7 +79,13 @@
           "title": "an ISO 8610 compatible string denoting a date and time",
```

Comment:
Do not modify this file. Follow the instructions here https://github.com/finos/datahelix/blob/master/docs/developer/HowToAddSupportForNewSchemaVersion.md to add a 0.2 schemaVersion instead